contrib/testnet/testnet.sh

```diff
@@ -271,7 +272,14 @@ set_bins()
     mount_repo_bins="-v ${KOVRI_REPO}/build/kovri:/usr/bin/kovri \
       -v ${KOVRI_REPO}/build/kovri-util:/usr/bin/kovri-util"
 
-    read_bool_input "Build repo binaries from within the container?" KOVRI_BUILD_REPO_BINS "Exec make release-static"
+    build_cmd="make clean"
```

Comment:
I'm not sure this is necessary. `make release` and `make release-static` include rules for making `util`, but I can add this in case those recipes change.